packages/shared/src/components/cards/PostCardFooter.tsx

```diff
@@ -50,7 +50,7 @@ export const PostCardFooter = ({
           )}
           loading="lazy"
           data-testid="postImage"
-          wrapperClassName="my-2"
+          {...(isVideoType && { wrapperClassName: 'overflow-hidden' })}
```

Comment:
Was this supposed to be changed to `'overflow-hidden'` 😬

Reply:
Damn, bad copy paste, thanks!